grantor.go

```diff
@@ -35,8 +36,8 @@ func (g *TokenGrantor) GrantToken(method string, claims *UserClaims) (string, er
 	if claims == nil {
 		return "", ErrNoClaims
 	}
-	if g.TokenSecret == "" {
+	if g.TokenHSSecret == "" {
```

Comment:
@njones , I don't want to change `TokenSecret` to `TokenHSSecret` because it makes reading this hard. Let's keep `TokenSecret`. Alternatively, I would say "TokenSharedSecret" (HS) as opposed to "TokenKeyPairSecret" (RS). I guess the focus is on the readability.

Reply:
Sure, I'll revert this... I was changing this to HS because (once again the JWT spec uses `HS`) and I can see adding the `ES` and `PS` keys and secrets as well, so naming this with HS makes it clear what values are expected for this variable.